features/step_definitions/secrets.js

```diff
@@ -79,6 +79,8 @@ module.exports = function server() {
         }).then((response) => {
             this.jobId = response.body[0].id;
             this.secondJobId = response.body[1].id;
+            if (2 in response.body) this.thirdJobId = response.body[2].id;
```

Comment:
Is response.body an array? That seems weird to me. 

You definitely shouldn't be using the `in` operator here, it doesn't always do what you expect it to do.
For example:
```
const a = ['a', 'b', undefined];
console.log(2 in a); // true
```

Instead, look for what you expect to find.
```
this.thirdJobId = typeof response.body[2] === 'object' ? response.body[2].id : null;
```

Also, always use brackets with if statements.